kotlin/internal/jvm/external_java_utils.bzl

```diff
@@ -0,0 +1,110 @@
+# Copyright 2018 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Various Java utils that were copied from pre-alpha branch of rules_android repo
```

Comment:
Any reason we can't just reference the ones in rules_android? Are they not public? Alternatively could we advocate to move these into https://github.com/bazelbuild/bazel-skylib?